.github/workflows/galaxy.yml

```diff
@@ -1,6 +1,6 @@
 ---
 name: Ansible Galaxy import
-on:
+"on":
```

Comment:
Why are these quoted?

Reply:
Was the linter interpreting them as truthy?

Reply:
Yup